shared/bundle_analysis/parsers/v3.py

```diff
@@ -0,0 +1,424 @@
+import json
```

Comment:
This entire file is basically a carbon copy of v2, except what is identified below. Note that each time a new stats file json schema  is updated we create a new parser version 95% of the code between each version would be the same.